jetty-servlets/src/test/java/org/eclipse/jetty/server/handler/gzip/GzipContentLengthTest.java

```diff
@@ -54,7 +54,7 @@
     private static final int LARGE = defaultHttp.getOutputBufferSize() * 8;
     private static final int MEDIUM = defaultHttp.getOutputBufferSize();
     private static final int SMALL = defaultHttp.getOutputBufferSize() / 4;
-    private static final int TINY = GzipHandler.DEFAULT_MIN_GZIP_SIZE / 2;
+    private static final int TINY = GzipHandler.BREAK_EVEN_GZIP_SIZE / 2;
```

Comment:
DEFAULT_MIN_GZIP_SIZE is a public API, don't change it for this version of Jetty.